### PR TITLE
feat(i18n): missing keys in settings

### PR DESCRIPTION
**Pull request by mininaim**

### CODE REVIEW

**Comment by jpkhawam** [comment]
@patak-dev looks good 👍🏼 you can merge

**Comment by jpkhawam** [comment]
@mininaim if you want, you can join the discord and we can have a channel for arabic i18n

**Comment by mininaim** [comment]
> @mininaim if you want, you can join the discord and we can have a channel for arabic i18n

Hey, the proper use of Arabic and localization within the locales file, still need improvements, specifically with the need for updates on certain words for Arabic users.

Thank you for the suggestion! I would be happy to join Discord and help out with with the Arabic i18n channel.

**Comment by jpkhawam** [comment]
@mininaim I'd certainly appreciate your input. It was a little hard for me to find all the appropriate words to use

**Comment by userquin** [comment]
the leading space is required?

imagen

**Comment by mininaim** [comment]
> ~the leading space is required?~
> 
> remove space before comma
> 
> imagen


Which line? because a word needed a space

File: locales/ar-EG.json [hunk 1]
```diff
@@ -207,49 +207,56 @@
...
   },
   "settings": {
     "about": {
-      "label": "بشأن Elk"
+      "label": "حول Elk",
+      "meet_the_team": "تعرف على فريقنا",
+      "sponsor_action": "تمويل Elk",
+      "sponsor_action_desc": "لتمويل تطوير Elk والمساعدة في تحسينه",
+      "sponsors": "الرعاة",
+      "sponsors_body_1": "تم تمويل Elk من قبل الشركات والأفراد التاليين:" ,
```
Comment (shuuji3) [inline locales/ar-EG.json:242]: Hello, @mininaim you can just remove the space just before `,` at the end of the line:
```suggestion
      "sponsors_body_1": "تم تمويل Elk من قبل الشركات والأفراد التاليين:",
```

**Review by shuuji3 — CHANGES_REQUESTED** [review]

**Comment by mininaim** [comment]
Apologies, I was using a formatter and I assumed it would automatically
remove the extra space after the comma.

**Comment by patak-dev** [comment]
See https://github.com/elk-zone/elk/pull/1217

**Pull request closed without merging**